src/data/nat/basic.lean

```diff
@@ -311,6 +323,9 @@ protected theorem div_le_div_right {n m : ℕ} (h : n ≤ m) {k : ℕ} : n / k 
 (nat.eq_zero_or_pos k).elim (λ k0, by simp [k0]) $ λ hk,
 (le_div_iff_mul_le' hk).2 $ le_trans (nat.div_mul_le_self' _ _) h
 
+lemma lt_of_div_lt_div (m n k : ℕ) : m / k < n / k → m < n :=
+λ h₁, by_contradiction $ λ h₂, absurd h₁ (not_lt_of_ge (nat.div_le_div_right (not_lt.1 h₂)))
```

Comment:
Minor point, `lemma lt_of_div_lt_div {m n k : ℕ} (h : m / k < n / k) : m < n :=` and without the lambda on the first line of the proof is preferred here.